### ISSUE TITLE
1000 log line default results set limitations.

### ISSUE THREAD

**Issue by zizzpudding** [grafana/explore-logs#919]
Would it be possible to extend the search past the default 100 log line limit without having to add more criteria? 

It would make it easier if we users could limit the result set to a smaller number of log lines, for instance 200 and an easy way to look at the next 200 or prev 200.  

I'm not sure if this is possible in Explore Logs.

**Comment by gtk-grafana** [comment]
@zizzpudding yeah this is a good change, do you want to mock up where we would put the line count limit in the UI?
In explore classic we tuck this away in the query options:
![Image](https://github.com/user-attachments/assets/e77a2db3-d2ba-4520-b6e5-eb3a44fc3a7c)


As to pagination I might argue that we should add infinite scroll support instead of diverging the UX for viewing more log lines then the limit. @matyax what do you think?

**Comment by matyax** [comment]
Yeah, agree. I'd like to test a combination of infinite scrolling with visualizing in logs volume what we call "visible range*" in classic Explore.

* Visible range is the range between the first and last log lines, <= the current requested interval.
